modules/nf-core/cnvkit/batch/environment.yml

```diff
@@ -1,8 +1,10 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
 channels:
   - conda-forge
   - bioconda
 
 dependencies:
   - bioconda::cnvkit=0.9.10
-  - bioconda::htslib=1.17 # Matched with the container
-  - bioconda::samtools=1.17 # Matched with the container
+  - bioconda::htslib=1.17
+  - bioconda::samtools=1.17
```

Comment:
we hsould bump samtools as well. How do we want to deal with all the versions of all mulled modules? We are not very consistent in either direction with version reporting I think